about2.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/style/main.css">
+    <link rel="stylesheet" href="/style/about2.css">
+    <title>Global Drummers Workshop</title>
+</head>
+
+<body class="font">
+    <header class="nav-container">
+        <div class="nav-socials-top">
+            <ul class="nav-socials">
+                <li>
+                    <a href="#"><i class="fa-brands fa-facebook-f"></i> </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <i class="fa-brands fa-twitter"></i>
+                    </a>
```

Comment:
- [ OPTIONAL] Try to connect you about styling in your index.html so that after running the page and clicking on about navigation, I can see the styled page.  
![image](https://user-images.githubusercontent.com/43877689/175391807-cb4f9cd9-ecbe-4860-9e87-fd0c542a658a.png)